apps/webcomponents/src/app/components/gn-search-input/gn-search-input.component.ts

```diff
@@ -14,4 +16,20 @@ import { BaseComponent } from '../base.component'
   encapsulation: ViewEncapsulation.ShadowDom,
   providers: [SearchFacade, SearchService],
 })
-export class GnSearchInputComponent extends BaseComponent {}
+export class GnSearchInputComponent extends BaseComponent {
+  @Input() datahubUrl: string
+
+  search(any: string) {
+    if (this.datahubUrl) {
+      const landingPage = `${this.datahubUrl}/home/search/?q=${any}`
+      window.open(landingPage, '_self').focus()
+    }
```

Comment:
Thinking about it more, it feels like this creates a strong dependency on the datahub app, whereas we could just offer more generic parameters like so:
```
open-on-search="../datahub/home/search/?q=${search}"
open-on-select="../datahub/dataset/${uuid}"
```
It's a bit more cumbersome to setup but much more flexible. One downside is that if the URL scheme of the datahub changes, people will have to manually change the configuration of their web components.

What do you think?